Client/mods/deathmatch/logic/CDeathmatchObject.cpp

```diff
@@ -89,7 +89,12 @@ void CDeathmatchObject::_StopMovement(bool a_bUnregister)
         delete m_pMoveAnimation;
         m_pMoveAnimation = NULL;
```

Comment:
`nullptr` instead of `NULL`

Reply:
This is not my code. There are a lot of NULLs throughout the file, as well as throughout the project. Better to do a separate PR with refactoring

Reply:
> This is not my code. There are a lot of NULLs throughout the file, as well as throughout the project. Better to do a separate PR with refactoring

Well, you are changing the file already so the change would not be significant

Reply:
Okay, so let's do it